Makefile

```diff
@@ -305,6 +332,10 @@ e2e-helm-deploy-release:
 e2e-kind-cleanup:
 	kind delete cluster --name kind
 
+.PHONY: e2e-eks-cleanup
+e2e-eks-cleanup:
+	eksctl delete cluster --name $(EKS_CLUSTER_NAME) --region $(AWS_REGION)
```

Comment:
Do our integration tests run serially? It seems like EKS_CLUSTER_NAME is defined statically so I think there may be a conflict on multiple concurrent runs.

Maybe there is some test UUID we can get from https://github.com/kubernetes/test-infra/pull/22150 to pass as a parameter to `make` to populate `EKS_CLUSTER_NAME`?

Reply:
> Do our integration tests run serially? It seems like EKS_CLUSTER_NAME is defined statically so I think there may be a conflict on multiple concurrent runs.

Yeah, this could be an issue in concurrent runs. The cluster name needs to be unique. Also, what is the cluster limit associated with this account used for testing? Wondering if there would be any impact when there are multiple runs of CI for PRs.

Reply:
The eksctl create cluster command creates an AWS VPC along with the eks cluster. I know AWS accounts have a limit of 5 VPCs per region but I think this limit can be updated for individual accounts

Reply:
I generate a UUID in the script and append that onto the cluster name